app/src/main/java/fr/free/nrw/commons/location/LocationServiceManager.java

```diff
@@ -50,7 +50,7 @@ public LocationServiceManager(Context context) {
      * @return true if the GPS provider is enabled
```

Comment:
Please can you update the Javadoc to simply say 'location provider' rather than 'GPS provider'